### ISSUE TITLE
[IA refresh] Moving wireframes to new live environment (part I: most pages)

### ISSUE THREAD

**Issue by taisdesouzalessa** [mozilla/foundation.mozilla.org#4724]
Follow up on: #4671 

### Background
Now that we have a new live environment: https://ia-refresh.mofostaging.net it is time to move the [current wireframes](https://mozilla.invisionapp.com/d/main?origin=v7#/console/19976285/418292231/preview?scrollOffset=0) to it. 

**Note: This is a _first draft_ only. These will not be the _final_ pages. They will be used to help determine what is needed, to get final copy, and approvals.**

### Questions
1. Do we have any pages at risk of major structural changes? If so, list them here:
[name of the page - estimated date on when we will have an answer on final page structure]
2. What we do > Press Center page doesn't have a mockup. How can we unblock that?
3. What are the conditions for the "What we fund > News" page? Are we filtering by hashtags/by keywords? 
4. What you can do has no mockup. Assuming it will be our current [Participate Page](https://foundation.mofostaging.net/en/participate/)? 

@kristinashu @beccaklam I need your help for this section. I feel the "who we are" pages are in the category "pages at risk of major structural changes" but I would love to hear you thoughts. 

### List of pages
Here are the list of pages that needs to be built:
- [ ] ~Home page~: blocked by the need of new components' development + executive feedback (ticket here #4727)
- [ ] ~Who we are page groups~: blocked by content (ticket here  #4725)
- [x] What we do > Overview (Taís)
- [x] What we do > Empower Action (Taís)
- [x] What we do > Connect Leaders (Taís)
- [x] What we do > Investigate and Research (Taís)
- [x] What we do > Press Center (Taís)
- [x] What we fund > Overview (Becca)
- [x] What we fund > Fellowships (Becca)
- [x] What we fund > Awards (Becca)
- [x] What we fund > Partners (Becca)
- [x] What we fund > Directory (Becca)
- [x] What we fund > Meet the Team (Becca)
- [x] What we fund > News: pre-work - make sure all news that should show up here are correctly tagged.  (Becca)
- [x] What you can do: no changes needed, right now
- [x] Blog: no changes needed

### Current content: 
https://docs.google.com/document/d/19Y5hNmEU4wix_gCoSvmhMKUk2F1Dht6Nwf1JQAu0lsg/edit?usp=sharing

### Notes:
As we look closely at these pages while we build them, we can [add notes here](https://docs.google.com/document/d/1DpjB_rqJ-Y47UdNFnjKGVAX_e6tuD_fRTSdWl_iRcTU/edit?usp=sharing) on things we should consider improve or if we notice any bugs

### Related tickets:
part II: #4725 | part III: #4727

**Comment by kristinashu** [comment]
> Do we have any pages at risk of major structural changes? If so, list them here:
    [name of the page - estimated date on when we will have an answer on final page structure]

For phase 1, the homepage should be the only one to require major changes. Aiming for design to be done end of June and dev to be done mid-july.

> What we do > Press Center page doesn't have a mockup. How can we unblock that?

Curious if we think it is useful? I think it helps highlight our impact and that partners would be interested in it. @beccaklam if you think it would be good to have then could you please mock it up as well? If we are unsure then we can ask for Anil's input.

> What are the conditions for the "What we fund > News" page? Are we filtering by hashtags/by keywords?

It is the Fellowships & Awards category.

> What you can do has no mockup. Assuming it will be our current Participate Page?

Correct!

**Comment by beccaklam** [comment]
> > What we do > Press Center page doesn't have a mockup. How can we unblock that?
> 
> Curious if we think it is useful? I think it helps highlight our impact and that partners would be interested in it. @beccaklam if you think it would be good to have then could you please mock it up as well? If we are unsure then we can ask for Anil's input.
> 
I put this in because Kevin put it in the WIP content doc. We can ask him for content or to see what he was thinking? I don't think it's a huge blocker, it's also something we can add later.

**Comment by kristinashu** [comment]
After reading the copy it sounds like it is a resource **for** press. Seems like it should be a link in the footer or on the about pages rather than a sub-page of What we do page.

**Comment by taisdesouzalessa** [comment]
What we do pages done. Missing: press page - figure out best placement for it.

**Comment by beccaklam** [comment]
"What we fund" pages are also done :)

**Comment by kristinashu** [comment]
This is very exciting and helpful to see! What is left to do?

**Comment by taisdesouzalessa** [comment]
Design part of "Press Center" page is here:  #4766.

**Comment by taisdesouzalessa** [comment]
The goal of this ticket was to move the wireframes to those pages. It was achieved. I wonder if we can close this ticket? @kristinashu @beccaklam? My suggestion is to open another ticket  (deadline, the following sprint) for:

- [ ] executive feedback (for all pages, including home and "who we are")
- [ ] content update and approval (for all pages, including home and "who we are")
- [ ] adjustments on the pages based on Treejack test
- [ ] UI design
- [ ] file bugs and wishlist for pages (to be discussed and prioritized tomorrow - June 09).

What do you think?

**Comment by kristinashu** [comment]
Thanks Tais! Yeah, I think this is good to close!

There is a ticket for the "Who We Are" page updates which should include both updates from exec feedback and content https://github.com/mozilla/foundation.mozilla.org/issues/4668.

The homepage is only the cause statement so I think that can be included as part of the design, there isn't really much we can do.

Could open a ticket for follow up on the Treejack test but we could also wait and see what the feedback is first.

UI design tickets would be good!

Could open a ticket for writing the wishlist, prioritize it, review with devs, and to then open implementation tickets.

**Comment by taisdesouzalessa** [comment]
Thanks for the guidance here, Kristina. I will leave it open for now until I create all the tickets requested here, okay?

**Comment by taisdesouzalessa** [comment]
RE: _"Could open a ticket for writing the wishlist, prioritize it, review with devs, and to then open implementation tickets."_
**Ticket here:** #4768

**Comment by taisdesouzalessa** [comment]
Content ticket for all pages except "who we are": #4771
UI ticket: #4772
